lib/structures/constants.js

```diff
@@ -9,4 +9,7 @@ module.exports = {
   FILE_OPEN_IF: 0x00000003,
   FILE_OVERWRITE: 0x00000004,
   FILE_OVERWRITE_IF: 0x00000005,
+
+  // Where does it come from?!
```

Comment:
This value was provided as is when I discovered the package, I did not question it.

Reply:
Yes, that's what I thought, but I think it makes sense to keep the comment as long as it's unclear where this magic number comes from, don't you agree?

Reply:
I agree.

Reply:
FYI: https://msdn.microsoft.com/en-us/library/cc246502.aspx

Reply:
Can you explain a bit more?

I guess it's the max size of a request packet minus the other fields, but i can't find it in the page :confused: